.github/workflows/test.yaml

```diff
@@ -12,3 +12,4 @@ jobs:
       - run: |
           npm install
           npm test
+          npm list
```

Comment:
Shouldn't it be placed before `npm test`? If the tests fail, doesn't that stop the run early, causing the needed package version listing to not happen?

Reply:
yeah woops, I'll move.